docs/en/ingest-management/fleet/fleet-manage-agents.asciidoc

```diff
@@ -14,6 +14,8 @@
 collect data and send it to the {stack}. Behind the scenes, {agent} runs the
 {beats} shippers or {elastic-endpoint} required for your configuration.
 
+//REVIEWERS: Is Elastic Endpoint correct here? ^^
```

Comment:
This looks like a dup of the doc `docs/en/ingest-management/elastic-agent/install-fleet-managed-agent.asciidoc`. What is the difference between the two documents?

Reply:
The files aren't duplicates. One describes how to install Elastic Agent; the other describes actions you can perform in Fleet to manage agents. (Right now, we have a separate installation section that describes all the different ways to install Elastic Agent, though that structure is likely to change later and is outside of the scope of this PR.

BTW, I do have another open that removes some of the redundant descriptions.